src/main/scala/uk/gov/hmrc/ct/computations/CPQ17.scala

```diff
@@ -41,5 +41,6 @@ case class CPQ17(value: Option[Boolean]) extends CtBoxIdentifier(name = "Trading
 }
 
 object CPQ17 {
-  val lossReform2017 = LocalDate.parse("2017-04-01")
+  def lossReform2017Applies(to: EndDate): Boolean = to.value.isAfter(lossReform2017)
```

Comment:
not sure I like to as a parameter name.

Why not call it endDate? Or even apEndDate?